src/components/SelectionList/Search/TransactionListItemRow.tsx

```diff
@@ -81,10 +81,13 @@ function ReceiptCell({transactionItem}: TransactionCellProps) {
 
     const backgroundStyles = transactionItem.isSelected ? StyleUtils.getBackgroundColorStyle(theme.buttonHoveredBG) : StyleUtils.getBackgroundColorStyle(theme.border);
 
-    const filename = getFileName(transactionItem?.receipt?.source ?? '');
-    const receiptURIs = getThumbnailAndImageURIs(transactionItem, null, filename);
-    const isReceiptPDF = Str.isPDF(filename);
-    const source = tryResolveUrlFromApiRoot(isReceiptPDF && !receiptURIs.isLocalFile ? receiptURIs.thumbnail ?? '' : receiptURIs.image ?? '');
+    let source = transactionItem?.receipt?.source ?? '';
```

Comment:
because you're doing `if(source)` below, then you can drop this part
```suggestion
    let source = transactionItem?.receipt?.source;
```